src/main/java/io/github/moulberry/notenoughupdates/ItemPriceInformation.java

```diff
@@ -130,6 +131,17 @@ public static void addToTooltip(List<String> tooltip, String internalname, ItemS
 			shiftStackMultiplier = stack.getTagCompound().getInteger(STACKSIZE_OVERRIDE);
 		}
 		int stackMultiplier = 1;
+		for (int i = 3; i < tooltip.size(); i++) {
+			String line = StringUtils.cleanColour(tooltip.get(i));
+			if (line.startsWith("Stored:")) {
+
+				String[] sackTooltip = line.replace("Stored: ", "").split("/");
+				if (!(Integer.parseInt(sackTooltip[0]) == 0)){
+
+					shiftStackMultiplier = Integer.parseInt(sackTooltip[0]);
+				}
+			}
```

Comment:
Please use a regex for this to avoid potential number format exceptions. Also try to avoid parsing the int twice here. Please also be aware that numbers like `20,145`  exist.